epsagon_kubernetes_exporters_deploy.sh

```diff
@@ -0,0 +1,93 @@
+#! /bin/bash
+function is_positive_answer {
+    answer=$1
+    if [ ${answer} == 'y' ] ; then
+        answer='Y'
+    fi
+    if [ ${answer} == 'Y' ] ; then
+        return 0;
+    fi
+    return 1;
+}
+
+function install_exporter {
+    CONTEXT=$1
+    CONFIG=$2
+    EXPORTER_FILENAME=$3
+    KUBECTL="kubectl --context ${CONTEXT} --kubeconfig=${CONFIG}"
+    ${KUBECTL} apply -f ${EXPORTER_FILENAME}
```

Comment:
Can we get an indication if the operation was successful or failed?

Reply:
If the operation failed then the user will see it locally.
If it succeeded - then we will be able to scrap the metric by our prometheus. We can (in addition) send a success message to a new endpoint at Epsagon - but its a design question we should discuss :)